operator/pkg/test_samples/kustomize/crd.yaml

```diff
@@ -1,20 +1,20 @@
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
-  name: samplecrds.operator.kyma-project.io
+  name: kustomcrds.operator.kyma-project.io
```

Comment:
this was done to make kustomize installable CRDs mutually exclusive from Helm repo and OCI installations